### ISSUE TITLE
Saka isn't reliably focused on Firefox

### ISSUE THREAD

**Issue by eejdoowad** [lusakasa/saka#8]

**Comment by 3v1n0** [comment]
True... One example is when you first `Ctrl+L` to higlight the tab bar, then you hit the saka combination, that won't move the focous to saka input field.

**Comment by azzamsa** [comment]
Sure. so many feature missing.

**Comment by pureooze** [comment]
So I have looked into this and it does not seem easy to fix. In particular, when we are dealing with trying to take focus away from the omnibar, it seems like the browser itself will prevent an extension from doing this. There is a workaround involving a hacky solution where the extension deletes the existing tab and clones it into a new instance of the tab. I would prefer to solve this using a more elegant solution than that but it does not seem to be easy to accomplish.

I'm going to instead try and focus on scenarios where the user is simply focused on an input inside a webpage (ex. the search on github) and then opens Saka.

**Comment by steinkim** [comment]
How about using windows.create api instread of injecting <iframe> or creating new tab to toggle saka?

Like this:

```javascript
  var w = WIDTH;
  var h = HEIGHT;
  var left = (screen.width / 2) - (w / 2);
  var top = (screen.height / 2) - (h / 2);

  browser.windows.create({
    'url': '/saka.html',
    'type': 'popup',
    'width': w,
    'height': h,
    'left': left,
    'top': top
  });
```

**Comment by pureooze** [comment]
Hi @steinkim, thanks for looking into this. I'm not sure how I feel about using a popup like this. On the one hand it does address some issues we have but at the same time I really don't like seeing the title bar.

**Comment by steinkim** [comment]
@pureooze https://chrome.google.com/webstore/detail/searchbar/fjefgkhmchopegjeicnblodnidbammed
I'm trying manipulating chrome extension default popup location like this chrome extension(after work).
